pywinauto/controls/atspi_controls.py

```diff
@@ -499,22 +499,22 @@ def __init__(self, elem):
         self.value = self.element_info.get_value()
 
     # -----------------------------------------------------------
-    def get_min_value(self):
+    def min_value(self):
         """Get the minimum value of the ScrollBar"""
         return self.value.get_minimum_value()
 
     # -----------------------------------------------------------
-    def get_max_value(self):
+    def max_value(self):
         """Get the maximum value of the ScrollBar"""
         return self.value.get_maximum_value()
 
     # -----------------------------------------------------------
-    def get_min_step(self):
+    def min_step(self):
         """Get the minimum step of the ScrollBar"""
         return self.value.get_minimum_increment()
 
     # -----------------------------------------------------------
-    def get_current_value(self):
+    def value(self):
```

Comment:
It conflicts with line 499! Attribute is overridden.

Reply:
Also `get_something` is a typical style in other backends. I don't see any reason to remove only some `get_` prefixes. Please keep it as `get_value` etc.

Reply:
`def value(self)` is present only in `uia_controls.SliderWrapper`. Let's keep this one. And use `get_value` in other wrappers.

Reply:
Still not renamed back to `get_value`. Why are you not reading the comments?